PNPM_SCRIPTS.md

```
@@ -1,118 +1,113 @@
-These scripts perform _only_ the deployment step, assuming the project has already been built (e.g., by `pnpm run build`). These are primarily used by the Cloudflare automated deployment system.
+- `pnpm run deploy:prod`
+  Builds and deploys to the production environment.
 
-- `pnpm run deploy:opennext:dev`
-  Deploys the built worker (`.open-next` directory) to the `allthingslinux-development` Cloudflare Worker service, using configuration from `wrangler.dev.jsonc`.
+- `pnpm run deploy`
+  Alias for production deployment - the default deploy command.
 
-- `pnpm run deploy:opennext:prod`
-  Deploys the built worker (`.open-next` directory) to the `allthingslinux-production` Cloudflare Worker service, using configuration from `wrangler.production.jsonc`.
+## Testing & Preview
```

markdownlint-cli2 (0.18.1)

Multiple headings with the same content

(MD024, no-duplicate-heading)